packages/grafana-ui/src/components/Monaco/types.ts

```diff
@@ -42,6 +42,8 @@ export interface CodeEditorProps {
   /** Handler to be performed when editor is blurred */
   onBlur?: CodeEditorChangeHandler;
 
+  onChange?: (val: string) => void;
```

Comment:
For consistency with the other "on..." functions, this should have a comment with a small description and be of the `CodeEditorChangeHandler` type.

Reply:
Done! [4fd2805](https://github.com/grafana/grafana/pull/57199/commits/4fd280585062df92a6fd01269257cb1860623667)